tracer/src/Datadog.AutoInstrumentation.NativeLoader/instrumented_assembly_generator/instrumented_assembly_generator_helper.h

```diff
@@ -0,0 +1,330 @@
+#pragma once
+#include "../com_ptr.h"
+#include "../cor_profiler.h"
+#include "../log.h"
+#include "../util.h"
+#include "instrumented_assembly_generator_consts.h"
+#include "method_info.h"
+#include "../../../../shared/src/native-src/dd_filesystem.hpp"
+#include <fstream>
+#include <random>
+#include <iomanip>
+#include <utility>
+#if !_WIN32
+#include <unistd.h>
+#endif
+
+using namespace shared;
+
+namespace instrumented_assembly_generator
+{
+template <typename T>
+shared::WSTRING IntToHex(T i)
+{
+    std::stringstream stream;
+    stream << "0x" << std::setfill('0') << std::setw(sizeof(T) * 2) << std::hex << i;
+    return ToWSTRING(stream.str());
+}
+
+inline fs::path instrumentedAssemblyGeneratorOutputFolder;
+
+inline std::string AnsiStringFromGuid(const GUID& guid)
+{
+#ifdef _WIN32
+    wchar_t szGuidW[40] = {0};
+    char szGuidA[40] = {0};
+    if (!StringFromGUID2(guid, szGuidW, 40))
+    {
+        Log::Warn("AnsiStringFromGuid: Failed in StringFromGUID2.");
+    }
+    WideCharToMultiByte(CP_ACP, 0, szGuidW, -1, szGuidA, 40, nullptr, nullptr);
+    return szGuidA;
+#else
+    throw std::logic_error("InstrumentationVerification on a non-Windows OS not yet implemented");
+#endif
+}
+
+// Helper method that removes all illegal chars from a file name
+// https://stackoverflow.com/a/31976060
+inline shared::WSTRING GetCleanedFileName(const shared::WSTRING& fileName)
+{
+    shared::WSTRING retFileName;
+#if _WIN32
+    const shared::WSTRING illegalChars = WStr("\\/:?\"<>|*");
+#else
+    const WSTRING illegalChars = WStr("/");
```

Comment:
prefix `WSTRING` by `shared::`
or remove `shared::` in the whole file (since you use `using namespace..`

Reply:
Fixed in ebde2f57c8ef03fd167862f981abd5c4958c222a